modules_extra/notifications.js

```diff
@@ -106,15 +110,15 @@ exports.create = function (api) {
   }
 
   function getFirstMessage(feedId, cb) {
-    api.sbot_user_feed({id: feedId, gte: 0, limit: 1})(null, cb)
+    api.sbot.user_feed({id: feedId, gte: 0, limit: 1})(null, cb)
```

Comment:
Yeah, it was something to do with not having nesting in depject 1. I never used snake case until I started using sodium. I kinda liked it there, and it showed that anything with snake case was close to the metal (c!) so, to be taken seriously. By the time it could be rewritten to be camel, when I was refactoring for depject2, I left it out because I just wanted to get it working, it's something that is actually quite a bit of work to change, but little real benefit.

I generally try to avoid camel names, unless there just isn't a horse name that suffices. (unless, like with directories, the prefix implies a distinction `isTest` (pure, returns boolean) `createThing` (returns a a thing with state)